graylog2-web-interface/src/views/components/visualizations/number/AutoFontSizer.tsx

```diff
@@ -26,10 +26,15 @@ import styled from 'styled-components';
 const TOLERANCE = 0.05;
 const CHILD_SIZE_RATIO = 0.8; // Proportion of the child size in relation to the container
 
-const FontSize = styled.div<{ fontSize: number }>`
+const FontSize = styled.div<{ fontSize: number, $center: boolean }>`
   height: 100%;
   width: 100%;
   font-size: ${(props) => `${props.fontSize}px`};
+  ${(props) => (props.$center ? `
```

Comment:
Let's use `css` from `styled-components` for the template string, this way `stylelint` and e.g. the css autocompletion will work properly.

Reply:
Thanks! :)